package.json

```diff
@@ -279,6 +281,7 @@
         "dependencies": [
           "@babel/core",
           "@babel/preset-typescript",
+          "@types/react-test-renderer",
```

Comment:
syncpack opt out behaviour as it doesn't understand single version policy and devDeps - note the version are the same in all packages having this specified as devDep

Reply:
If syncpack can be modified to help with that I'm happy to take a look if you'd like to open an issue  to explain this use case.

Reply:
hey @JamieMason , thanks for chiming in. Once this is merged and we'll have some extra time , we gonna submit an issue. cheers

Btw, just out of curiosity how did you managed to notice in a draft PR that there was mention on syncpack ? :D

Reply:
Great, sounds good 👍🏻 

> Btw, just out of curiosity how did you managed to notice in a draft PR that there was mention on syncpack ? :D

from [github code/issues search](https://github.com/search?o=desc&q=syncpack&s=updated&type=Issues) it's (admittedly kind of sad but also) a handy way to find out about problems or feature feedback 😄